src/utils/pricing.ts

```diff
@@ -4,34 +4,43 @@ import { exponentToBigDecimal, safeDiv } from '../utils/index'
 import { Bundle, Pool, Token } from './../types/schema'
 import { ONE_BD, ZERO_BD, ZERO_BI } from './constants'
 
-const WETH_ADDRESS = '0x4200000000000000000000000000000000000006'
-const USDC_WETH_05_POOL = '0x4c36388be6f416a29c8d8eee81c771ce6be14b18'
+// todo: use a wmatic pool? check which has more liq first
+const WETH_ADDRESS = '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619'
+const USDC_WETH_03_POOL = '0x0e44ceb592acfc5d3f09d996302eb4c499ff8c10'
```

Comment:
`0xa374094527e1673a86de625aa59517c5de346d32`

has more liquidity and uses matic as the native token 

USDC is token1

Reply:
updated this